src/libs/actions/Report.js

```diff
@@ -1232,10 +1232,11 @@ function deleteReportComment(reportID, reportAction) {
  *
  * @param {Number} reportID
  * @param {Number} [sequenceNumber] This can be used to set the last read actionID to a specific
+ * @param {Boolean} manuallyMarked If the user manually marked this as unread, we need to tell the API
```

Comment:
```suggestion
 * @param {Boolean} [manuallyMarked] If the user manually marked this as unread, we need to tell the API
```

Because it is optional